src/components/dialog/dialog/dialog-base-element.ts

```diff
@@ -0,0 +1,157 @@
+import { type PropertyValues } from 'lit';
+import { property } from 'lit/decorators.js';
+
+import { SbbFocusHandler } from '../../core/a11y.js';
+import { SbbOverlayBaseElement } from '../../core/base-elements.js';
+import { SbbLanguageController } from '../../core/controllers.js';
+import { hostContext, SbbScrollHandler } from '../../core/dom.js';
+import { EventEmitter } from '../../core/eventing.js';
+import { i18nDialog } from '../../core/i18n.js';
+import type { SbbDialogCloseEventDetails } from '../../core/interfaces.js';
+import { SbbNegativeMixin } from '../../core/mixins.js';
+import { applyInertMechanism, removeInertMechanism } from '../../core/overlay.js';
+import type { SbbScreenReaderOnlyElement } from '../../screen-reader-only.js';
+
+// A global collection of existing dialogs
+export const dialogRefs: SbbDialogBaseElement[] = [];
```

Comment:
Is that ok, from feature perspective, if we mix dialog and overlay in the same? Probably yes, but really? :D

Reply:
As discussed yesterday, i renamed everything to 'overlay<stuff>' as it's a more generic prefix.